daisy_workflows/image_build/debian/deb8.wf.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Name": "build-deb8",
```

Comment:
Sorry to nit here, deb has a different meaning than debian so we should spell it out in workflows (and workflow names)